twitchpopups.htm

```diff
@@ -149,6 +157,34 @@ <h1 id="popuptext"></h1>
         $("#popuptext").animate({"opacity":0, "margin-left":"50px"}, 700);
     }
     
+    const formatEmotes = (message, emotes, upperCase) => {
+        //parse the message for html and remove any tags
+      if(upperCase){
+        message = message.toUpperCase();
+      }
+      
+      let newMessage = $($.parseHTML(message)).text().split("");
+
+      //replace any twitch emotes in the message with img tags for those emotes
+      if(twitchEmotes){
+        for (let emoteIndex in emotes) {

```

Comment:
just to save somone else going through the pain you have, could you do 4 space indentation?

Reply:
Sorry really thought I caught this one.

Reply:
Does 9b85627 look right?

Reply:
Bang on :)